projects/hslayers/services/styler/styler.service.ts

```diff
@@ -749,10 +797,13 @@ export class HsStylerService {
         const qmlParser = new QGISStyleParser();
 
         await qmlParser.readStyle(styleString);
-        this.qml = styleString;
+        this.qml = styleString.replace(
+          /base64:/g,
+          'data:image/svg+xml;base64,',
```

Comment:
QMLs will include `source` attribute on both `Option` (new fromat)  and `prop` (necessary  backwards comaptiblity) elements containing base64. It can be then used to adjust the value as needed. 

For now we will parse via `prop` as Geostyler still takes the value from that attribute. See [02e923f](https://github.com/hslayers/hslayers-ng/pull/5008/commits/02e923fda08bb23d490c8df0d962e1119de0763b)